library/src/scripts/components/SelectBox.tsx

```diff
@@ -95,7 +96,9 @@ export default class SelectBox extends React.Component<ISelfLabelledProps | IExt
                     index={i}
                     current={selected}
                     lang={child.lang}
-                    buttonClassName="dropDownItem-button selectBox-buttonItem"
+                    buttonClassName={classNames("dropDownItem-button", "selectBox-buttonItem", {
+                        isInModal: this.props.openAsModal,
```

Comment:
Do we actually use this new class anywhere here? And if we do wouldn't it be better to use a CSS selector like `.modal .selectBox-buttonItem`?

Reply:
No, I want to cut down as much as is possible and reasonable to nest styles for nothing. It cuts down a LOT on CSS in the long run. Good example is the panel layout, I cut by about half the CSS by setting classes in React.